internal/impl/devbox.go

```diff
@@ -288,9 +291,9 @@ func (d *Devbox) NixEnv(ctx context.Context, opts devopt.NixEnvOpts) (string, er
 			)
 		}
 
-		envs, err = d.computeNixEnv(ctx, true /*usePrintDevEnvCache*/)
+		envs, err = d.computeDevboxEnv(ctx, true /*usePrintDevEnvCache*/)
 	} else {
-		envs, err = d.ensurePackagesAreInstalledAndComputeEnv(ctx)
+		envs, err = d.computeCurrentDevboxEnv(ctx)
```

Comment:
This is confusing. A reader can't tell what the difference between these two function calls is.

What about `ensureProjectStateIsCurrentAndComputeEnv`

Reply:
I don't like this one. I'll explain elsewhere.